src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
+  return response.json();
+}
```

Comment:
We are not handling errors, checking if our response is what we expected and not checking [response.ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) property. This function could be rewritten such as:
```ts
import { z } from "zod" // zod is a validation library, other popular choices are yup or joi

export const BASE_URL = 'https://mate.academy/students-api/';

const getAllUsersSchema = z.object({
    data: z.array(z.object({
        name: z.string()
    }))
})

export async function getAllUsers() {
    try {
        const response = await fetch(`${BASE_URL}/users`);
        if (!response.ok) {
            throw new Error("Failed to fetch all users!")
        }
        // If api returns somethings else than we defined in schema zod throws a validation erorr
        const safeUsers = getAllUsersSchema.parse(response.json())
    }
} catch (err) {
    console.log(err)
    // further error handling
}
```
Now we are ensuring that the function return type is correct and checking for any possible api errors.